src/main/webapp/app/exercises/quiz/manage/quiz-exercise-detail.component.ts

```diff
@@ -152,7 +161,10 @@ export class QuizExerciseDetailComponent implements OnInit, OnChanges, Component
         /** Initialize local constants **/
         this.showExistingQuestions = false;
         this.showExistingQuestionsFromCourse = true;
+        this.showExistingQuestionsFromExam = false;
+        this.showExistingQuestionsFromFile = false;
         this.courses = [];
+        this.exams = [];
```

Comment:
Same here, you already initialized that above

Reply:
👍